dotcom-rendering/src/web/lib/notification.ts

```diff
@@ -16,8 +18,13 @@ export const mapBrazeCardsToNotifications = (
 ): Notification[] => {
 	return cards.filter(hasTargetAndMessage).map((card) => {
 		return {
+			id: card.id,
 			target: card.extras.target,
 			message: card.extras.message,
+			logImpression: () => {
+				card.logImpression();
+				console.log('logged impression');
```

Comment:
can be removed?

Reply:
Oh how did I miss that?! I was about to merge, but will remove this first.